bot/cogs/todo_list.py

```diff
@@ -222,8 +223,8 @@ async def nuke_todos(self, ctx: Context) -> None:
         if TODO_FILE.is_file():
             if os.stat(TODO_FILE).st_size > 0:
 
-                with open(TODO_FILE, "r+") as read:
-                    todo_file_read = json.load(read)
+                async with aiofiles.open(TODO_FILE, "r+") as read:
+                    todo_file_read = json.load(await read.read())
```

Comment:
this needs to be `json.loads()`, not `json.load()`, I changed it on my end for the merge.